core/prototypes/module.py

```diff
@@ -146,3 +158,55 @@ def baseSkel(self, *args, **kwargs) -> SkeletonInstance:
         By default, baseSkel is used by :func:`~viewSkel`, :func:`~addSkel`, and :func:`~editSkel`.
         """
         return self._resolveSkelCls(*args, **kwargs)()
+
+    def describe(self) -> Dict:
+        """
+        Meta description of this module.
+        """
+        # Use cached description?
+        if isinstance(self._cached_description, dict):
+            return self._cached_description
+
+        # Retrieve handler
+        if not (handler := self.handler):
+            return None
+
+        if callable(handler):
+            handler = handler()
+
+        # Default description
+        ret = {
+            "name": self.__class__.__name__,
+            "handler": ".".join((handler, self.__class__.__name__.lower())),
+        }
+
+        # Extend indexes, if available
+        if indexes := getattr(self, "indexes", None):
+            ret["indexes"] = indexes
```

Comment:
```suggestion
        # Extend indexes, if available 
         ret["indexes"] = getattr(self, "indexes", [])
```
I would prefer to have an empty list here so that you always have a field to access in the frontend.

Reply:
Hello @ArneGudermann, I understand your concerns. On the other hand, any client relying on this structure should handle the case when something isn't there. Also, I'm of the opinion to keep unused information low, especially when its sent via network. I know, this is only an empty list, but it's an information which contains the same statement if it's not there.

Reply:
Okay, I understand that you just have to pay attention to it then I am so far satisfied with it.